### DISCUSSION TITLE
This is for synergy, Where is the one for DL Gen 10/ iLO5

### DISCUSSION THREAD

**Discussion by zeshan1977** [HewlettPackard/POSH-HPOneView#578] [Q&A]
https://github.com/HewlettPackard/POSH-HPEOneView/blob/6.10/Samples/Alerts_Sample.ps1

**Comment by ChrisLynchHPE** [comment]
You have a few options:

1. Change the `-HealthCategory` parameter to `server-hardware`, which will return all alerts for server hardware.
2. Get the server resource from `Get-OVServer` Cmdlet, then pass to `Get-OVAlert` either using a pipeline operation, or using `-InputObject` parameter value.
